### ISSUE TITLE
[BUG] Alt Text on banner image

### ISSUE THREAD

**Issue by InHuOfficial** [eddiejaoude/eddiejaoude#112]
## Issue
Alt text on the animated GIF collage at the top of the page is missing. 

This means that the link to http://eddiejaoude.io/ could read out the image URL as the link text in some screen reader / browser combinations, which is confusing.

### Image in question
![Eddie animated banner](https://github.com/EddieHubCommunity/Branding/raw/main/community/Eddie_banner_GIF.gif)

## Fix
Add `alt` text to the image that says "Eddie Jaoude Personal Site" or similar so the link purpose is obvious to screen reader / assistive tech users.

**Comment by eddiejaoude** [comment]
Thank you @InHuOfficial 👍
